src/components/FeatureTrainingModal.tsx

```diff
@@ -208,6 +208,7 @@ function FeatureTrainingModal({
     const [illustrationAspectRatio, setIllustrationAspectRatio] = useState(illustrationAspectRatioProp ?? VIDEO_ASPECT_RATIO);
     const {shouldUseNarrowLayout} = useResponsiveLayout();
     const {isOffline} = useNetwork();
+    const isHelpButtonPressed = useRef(false);
```

Comment:
It looks like this is never set to `false`, so perhaps a better name would be `hasHelpButtonBeenPressed`, indicating that it's not reflective of an ongoing pressed state, but rather a flag for whether it has been pressed at least once since the component has mounted